pkg/asset/machines/master.go

```diff
@@ -155,6 +155,8 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 	mign := &machine.Master{}
 	dependencies.Get(clusterID, installConfig, rhcosImage, mign)
 
+	masterUserDataSecretName := "master-user-data"
```

Comment:
I am happy to take these changes, but please include them in a separate PR.

Also, use a constant at the package scope rather than a variable at the function scope.

Reply:
Ah, sorry. I missed the part where this was used in the secret name at the end. Sounds good.